pravega/src/test/scala/docs/scaladsl/PravegaReadWriteDocs.scala

```diff
@@ -84,36 +89,37 @@ class PravegaReadWriteDocs {
 
     }
 
-  case class Person(id: String, firstname: String)
+  case class Person(id: Int, firstname: String)
```

Comment:
This class exists in `package akka.stream.alpakka.pravega`, as well. Might be easier with just one?